test/reducers/navigator.spec.js

```diff
@@ -56,4 +56,18 @@ describe('navigate reducer', () => {
     expect(state.stack.count()).to.be.equal(2);
     expect(state.stack.first()).to.be.equal('test');
   });
+
+  it('expect REPLACE_AT_INDEX to replace the route at the index passed', () => {
```

Comment:
If at some point we'll remove an invariant (for some reason) or it'll be a broken (always return `true`, for instance) - test will show it.